packages/commerce-sdk-react/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v3.2.0-dev (Oct 14, 2024)
```

Comment:
this version and date is likely wrong! What should I put here?

Reply:
leave it as it, we will figure it out during release :) thanks for adding changelog